sktime/dists_kernels/scipy_dist.py

```
@@ -78,16 +81,29 @@ def _transform(self, X, X2=None):
         """
         p = self.p
         metric = self.metric
+        weights = self.weights
 
         if isinstance(X, pd.DataFrame):
             X = X.select_dtypes("number").to_numpy(dtype="float")
 
         if isinstance(X2, pd.DataFrame):
             X2 = X2.select_dtypes("number").to_numpy(dtype="float")
 
+        if np.ndim(weights) == 1:
```

hm, you probably also want to raise an error if the weights vector is not 1D